cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -44,6 +44,10 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading *authorization* enabled CDAP instance you will need to give *cdap* user *ADMIN* privilege on
+   all existing CDAP Namespaces. Note: *ADMIN* privilege does not give *cdap* user any access (*READ/WRITE*)
```

Comment:
... does not give READ/WRITE privileges. 

(I think "any access" adds to confusion)